controllers/reconcilers/kserve_networkpolicy_reconciler.go

```diff
@@ -30,14 +31,19 @@ import (
 )
 
 const (
-	networkPolicyName = "allow-from-openshift-monitoring-ns"
+	monitoringNetworkPolicyName            = "allow-from-openshift-monitoring-ns"
+	openshiftIngressNetworkPolicyName      = "allow-openshift-ingress"
+	opendatahubNamespacesNetworkPolicyName = "allow-from-opendatahub-ns"
 )
 
+var definedNetworkPolicies = []string{monitoringNetworkPolicyName, openshiftIngressNetworkPolicyName, opendatahubNamespacesNetworkPolicyName}
+
 type KserveNetworkPolicyReconciler struct {
 	client               client.Client
 	scheme               *runtime.Scheme
 	networkPolicyHandler resources.NetworkPolicyHandler
 	deltaProcessor       processors.DeltaProcessor
+	appNamespace         string
```

Comment:
I need to know where this ctrl was deployed as part of RHOAI, as we need a policy allowing traffic from this ns - otherwise components like pipelines can fail. See this [comment](https://issues.redhat.com/browse/RHOAIENG-1003?focusedId=23691645&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-23691645)

Reply:
To fetch the value of ctrl namespace we could directly use `constants.KServeNamespace`. We could remove this extra function from utils.

Reply:
I can't find `constants.KServeNamespace` in the code base, but I wonder how constant can handle something which is dynamic in its nature. I mean it can be `rhods-applications` or `opendatahub` for example.

Reply:
`constants` is coming from "github.com/kserve/kserve/pkg/constants"

Reply:
Ah found it. `getEnvOrDefault("POD_NAMESPACE", "kserve")` so in fact a `var`, that's what got me confused :) 

If we can assume this env variable is always there that will do. Thanks for sharing.

Reply:
Fixed in 7e30338

Thanks for sharing, I didn't know about its existence. Only about the `POD_NAMESPACE` env variable as such.

Reply:
KServeNamespace                  = getEnvOrDefault("POD_NAMESPACE", "kserve")

Reply:
this env always initialize
```
          env:
            - name: POD_NAMESPACE
              valueFrom:
                fieldRef:
                  fieldPath: metadata.namespace
                  ```

Reply:
I'm no longer sure about the intended goal. Now, on the code another dev seein `constants.KServeNamespace` would think that it would return KServe controller namespace, while it is odh-model-controller namespace.

When installing via the operator, it is the same. But for development, it is not. Not sure if this will be confusing.
At least, drop a comment that such value would refer to odh-model-controller namespace, rather than KServe controller namespace.

Reply:
As an aside: we are due to update the KServe version in go.mod.